android/daemon/src/main/java/org/mozilla/firefox/vpn/daemon/BootReceiver.kt

```diff
@@ -8,6 +8,7 @@ import android.content.BroadcastReceiver
 import android.content.Context
 import android.content.Intent
 import android.os.Build
+import org.mozilla.firefox.qt.common.Prefs
```

Comment:
Why add these import if this file has no changes?

Reply:
The `Prefs` moved from `/daemon`to `/common` therefore i need to import it :)